app/perfdata.h

```diff
@@ -24,9 +24,11 @@
 #include "perffeatures.h"
 #include "perfheader.h"
 
+#include "config-perfparser.h" // generated by cmake
```

Comment:
The reason for the comment is because the next person who tries the qmake or qbs buildsystem will hit "config-perfparser.h: file not found" and will really wonder where it's supposed to come from. In such a case, there's no generated file to open....

Your call in the end, but I think the comment is useful due to the other buildsystems.
If we can delete the other buildsystems then it's simpler ;)